README.rst

```diff
@@ -79,8 +115,18 @@ Finding and configuring device name in **NI Hardware Configuration Utility**:
   :align: center
   :width: 800px
 
-Tasks in NI-DAQmx
------------------
+Python Examples
+===============
+
+You can find a variety of examples in the GitHub repository in the
+`nidaqmx-python examples <https://github.com/ni/nidaqmx-python/tree/master/examples>`_
```

Comment:
Based on experience from https://github.com/ni/measurement-plugin-python, I am hesitant to point users at the main/master branch, because the latest unreleased version of the examples may depend on features that are only in the latest unreleased source.

I recognize that nidaqmx-python doesn't change as quickly as measurement-plugin-python, so maybe a disclaimer about versioning is enough:

"For best results, use the examples corresponding to the version of **nidaqmx** that you are using. For example, if you are using version 1.0.0, check out the releases/1.0 branch or the 1.0.0 tag. Newer examples may demonstrate features that are not available in older versions of **nidaqmx**."

(BTW there is no releases/1.0 branch yet, but this is a reason to create one. We can also enable releases/1.0 or 1.0.0 tag in readthedocs...)

Reply:
Great idea. We already use tagging, so I'll mention that. For readthedocs, we have a stable version that follows the latest release tag. From https://docs.readthedocs.io/en/stable/versions.html:

> If your project has any tags or branches with a name following [semantic versioning](https://semver.org/), we also create a stable version, tracking your most recent release. If you want a custom stable version, create either a tag or branch in your project with that name.

So I'll update my links to readthedocs to go there as well.